### ISSUE TITLE
IMPORT FAILED

### ISSUE THREAD

**Issue by PGCRT** [WASasquatch/was-node-suite-comfyui#519]
ComfyUI: v0.3.7-3-g8af9a91
         (2024-12-06)
Manager: V2.55.4

Up to date, tried to uninstall reinstall and still got an import failed, could be related to a recent Comfy update that break things since this issue is more recent than your last commit,

**Comment by menahem121** [comment]
yes im experiencing the same issue it seems to be related to opencv-python-headless the package is automatically uninstalled at comfyUI launch to be replaced with the opencv-python package

**Comment by PGCRT** [comment]
I just reinstalled comfuyUI, seems that it fixed the issue for me, even if it's the same version and packages as before. so I can't really say what caused this failed import error

**Comment by WASasquatch** [comment]
May have been order of installation. While I have tried to unpin specific
versions so it's more compatible, other packages may pin lesser versions
themselves
